kickoff_meetings.md

```diff
@@ -0,0 +1,37 @@
+# Kickoff Meeting Outline
+This is a general outline of the kickoff meeting to help the meeting leads guide the meeting.
+
+## Introduction
+- Brief intro to mentorship.
+    - Process
+    - Duration
+    - Graduation
+- Explain the commitments expected from both the mentor and mentee during the program duration
+  - Mentorship diary: an issue to report interactions and milestones
+  - Joining WG meetings is required (if applicable)
+- The mentee and mentor can reach the mentorship team on slack or the Education Assistant (EA)
+  - Introduce the Education Assistant and explain his role.
+
+## Scope Introduction
+- The mentor should explain the scope they will be mentoring in
+
+## Goals & Expectations
+- The mentoring relationship should be *mentee-driven and mentor-guided*.
+- Mentor and mentee should discuss the mentee's goals and expectations
+- Mentees should drive activity towards career goals (have a “growth attitude”).
+- Mentors should help their mentees refine their goals during the first week of the program.
+- The mentor should mention their expectation reaching the end of the program.
+
+## Availability
+- Meeting dates and times
+- Medium of communication
+
+## Privacy & CoC
+- Review and confirm that all meetings and interactions will be done with respect to [Node.js Code of Conduct][]
+- Determine upfront what should be confidential and what can be disclosed, or designate a process such as having private sessions at end of meetings when needed.
+- Determine if the mentee and mentor are comfortable recording sessions
+
```

Comment:
I think everything below this point should be deleted. If you were wanting an extra link to the CoC besides the one above, the markup needs to be fixed to:

`[Node.js Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)`